kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1427,6 +1427,14 @@ object KyuubiConf {
     .timeConf
     .createWithDefault(Duration.ofSeconds(15).toMillis)
 
+  val ENGINE_ALIVE_MAX_FAIL_COUNT: ConfigEntry[Int] =
+    buildConf("kyuubi.session.engine.alive.max.fail.count")
```

Comment:
How about changing the config name to `kyuubi.session.engine.alive.max.failures` , following the style of existed config `kyuubi.engine.deregister.job.max.failures` ? The word `count` doesn't contribute much semantic meaning for definition or usage.